src/Uno.UI/UI/Xaml/UIElement.Pointers.wasm.cs

```diff
@@ -287,7 +253,7 @@ internal static void RemoveActivePointer(PointerIdentifier managedId)
 		const int exitOrUp = (int)(NativePointerEvent.pointerout | NativePointerEvent.pointerup);
 
 		var pointerType = (PointerDeviceType)args.deviceType;
-		var pointerId = TransformPointerId(new PointerIdentifier((Windows.Devices.Input.PointerDeviceType)pointerType, (uint)args.pointerId));
```

Comment:
I recall the cast was needed for WinUI builds. Let's see what CI will say though.